modules/ai_drivers/settings.hpp

```diff
@@ -0,0 +1,18 @@
+// Array containing vehicles that will have option to spawn AI driver in them
+// Example: selectVehicles[] = {"BluforLandRover1", "BluforLandRover2"};
+selectVehicles[] = {};
```

Comment:
Just curious for my own benefit, what's the reason / advantage of putting this in a `.hpp` file instead of `.sqf`?

Reply:
not having the GVAR macros for people to fuck up mainly, also inheritance options for certain things like side options, classes of objects settings, etc. Similar to how many mods do it via cfgVehicles